examples/Autopoint/generate_gaussian_mixture_data.py

```diff
@@ -0,0 +1,15 @@
+import math
```

Comment:
Should you have 1 file instead of two files for generating gaussian mixture data?
 Also, can you please add flags to the generation script so that the user can define them from the command line?

Reply:
the reason for the two separate files is because of the visualization part. I have taken out the visualization for generated data.